packages/linux-driver-addons/dvb/depends/media_tree/package.mk

```diff
@@ -50,4 +50,18 @@ unpack() {
     cp -a "$(kernel_path)/drivers/media/v4l2-core/videobuf-res.c" "$PKG_BUILD/drivers/media/v4l2-core/"
     cp -a "$(kernel_path)/include/media/videobuf-res.h" "$PKG_BUILD/include/media/"
   fi
+
+  # Copy avl6862 driver
+  if listcontains "$ADDITIONAL_DRIVERS" "avl6862-aml"; then
+    mkdir -p  "$PKG_BUILD/drivers/media/dvb-avl/"
+    cp -a $(kernel_path)/drivers/amlogic/dvb-avl "$PKG_BUILD/drivers/media/"
+    echo "obj-y += dvb-aml/" >> "$PKG_BUILD/drivers/media/Makefile"
+  fi
+
+  # Copy wetekdvb driver
+  if listcontains "$ADDITIONAL_DRIVERS" "wetekdvb"; then
```

Comment:
I know that WP1 doesn't have neither crazycat nor hauppauge enabled but it has "wetekdvb" listed in ADDITIONAL_DRIVERS:
https://github.com/LibreELEC/LibreELEC.tv/blob/master/projects/WeTek_Play/options#L115

So if you copy wetekdvb_play2.ko it will be the wrong module.

Reply:
I think there'll be good idea distinguish WP and WP2 drivers and rename it wetekdvb2.